services/ui-src/src/components/rates/types/NDREnhanced.tsx

```diff
@@ -74,7 +74,7 @@ export const NDREnhanced = (
   return (
     <Stack gap="2rem">
       <CmsdsTextField
-        label="Performance Rates Denominator"
+        label={(label ?? "Performance Rates") + " Denominator"}
```

Comment:
This is an instance where I could go either way on string interpolation vs concatenation. What do you think?

```suggestion
        label={`${label ?? "Performance Rates"} Denominator`}
```